deploy/crds/openebs.io_jivavolumepolicies_crd.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Are these separate files required, as you already have them included in the operator.yaml?

Reply:
These files are automatically generated by the operator sdk.